README.md

```diff
@@ -539,6 +539,33 @@ crud.len('customers')
 ...
 ```
 
+### Count
```

Comment:
Add some words about method itself. It is worth to mention a precision of returned value.

Reply:
Done:

```
### Count

`CRUD` supports multi-conditional count, treating a cluster as a single space.
The same as with `select()` the conditions may include field names or numbers,
as well as index names. The recommended first condition is a TREE index; this
helps reducing the number of tuples to scan. Otherwise a full scan is performed.
If compared with `len()`, `count()` method works slower because `count()`
scans the entire space to count the tuples.
This method does yield that's why result may be approximate.
```